lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -86,6 +87,30 @@ def test_savefig_to_stringio_with_usetex_eps():
     _test_savefig_to_stringio(format='eps')
 
 
+@cleanup
+def test_combine_images():
+    #Test that figures can be saved with and without combining multiple images
+    #(on a single set of axes) into a single image.
+    X, Y = np.meshgrid(np.arange(-5, 5, 1), np.arange(-5, 5, 1))
+    Z = np.sin(Y ** 2)
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1)
+    ax.set_xlim(0, 3)
+    ax.imshow(Z, extent=[0, 1, 0, 1])
+    ax.imshow(Z[::-1], extent=[2, 3, 0, 1])
+    plt.rcParams['vector_backends.combine_images'] = True
+    with io.BytesIO() as ps:
+        fig.savefig(ps, format="ps")
+        ps.seek(0)
+        buff = ps.read()
+        assert buff.count(' colorimage') == 1
```

Comment:
The failure appears to be here, and similar locations below and in the svg test.  I think `' colorimage'` needs to be bytes, not a string.  I think you can wrap it with `six.b()`.